### ISSUE TITLE
jira-cli new - provide labels

### ISSUE THREAD

**Issue by voigt** [alisaifee/jira-cli#18]
```
 jira-cli new -h
usage: jira-cli new [-h] [--jira-url JIRA_URL] [--format FORMAT] [--oneline]
                    [-v] [-u USERNAME] [-p PASSWORD] [--protocol {soap,rest}]
                    [--priority ISSUE_PRIORITY] [--project ISSUE_PROJECT]
                    [--description ISSUE_DESCRIPTION] [--type ISSUE_TYPE]
                    [--parent ISSUE_PARENT] [--assignee ISSUE_ASSIGNEE]
                    [--reporter ISSUE_REPORTER]
                    title
```
Hey,

according to `jira-cli new -h` its not possible to provide labels to the issue. Will this be supported anytime?

Kind Regards,
Christoph

**Comment by alisaifee** [comment]
Hi @voigt : I've added support for labels both for issue creation and updates. simply use `--label=foo --label=bar` etc. The feature is available in version `2.0.9`

**Comment by voigt** [comment]
Awesome, I was still on 2.0.8... Thx a lot! :)
